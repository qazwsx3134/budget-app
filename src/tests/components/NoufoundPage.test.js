import React from "react";
import { shallow } from "enzyme";
import NotFound from "../../components/NotFoundPage";

test('should not found page render right', () => {
    const wrapper = shallow(<NotFound />)
    expect(wrapper).toMatchSnapshot();
})
