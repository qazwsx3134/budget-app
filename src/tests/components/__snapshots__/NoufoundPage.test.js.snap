// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should not found page render right 1`] = `
<div>
  404 - 
  <Link
    to="/"
  >
    Go home
  </Link>
</div>
`;
